rdr_service/config.py

```diff
@@ -178,6 +178,12 @@
 NPH_SAMPLE_DATA_BIOBANK_NIGHTLY_FILE_DROP = 'nph_sample_data_biobank_bucket_name'
 CE_MEDIATED_HPO_ID='ce_mediated_hpo_id'
 
+GENOMICS_SITES_DATA_BUCKETS = {
+    "bcm": "prod-genomics-data-baylor",
+    "co": "prod-genomics-data-color",
+    "bi": "prod-genomics-data-color",
```

Comment:
This should be `prod-genomics-data-broad`.

Reply:
It should be `prod-genomics-data-broad` instead of `prod-genomics-data-color`?

Reply:
bi -> Broad Institute. If you just needs this to be CVL data buckets they maybe we can update the name. Coor sequences the BI samples for the CVL/WGS pipeline.

Reply:
Ah, ok yeah I just need the cvl data buckets, so I was mapping Broad to the Color bucket. I thought that bi & co were different names for the same cvl site. I will update the name, thank you!